src/pages/RadicalCollaboration/Share/MediaInfo.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import useStyles from './styles.js';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Link from '../../../components/common/Link';
+
+const CtiLogoSection = () => {
+    const classes = useStyles();
+  return (
+    <Container className={classes.logoContainer}>
+        <img className={classes.logoBW} src='/images/OneColor_CTI_Logo.svg' alt='CTI logo' />
```

Comment:
Might want to move this src path into a const variable and put it after const classes. That way it can be easily found and updated later if needed